### DISCUSSION TITLE
NLME - Nonlinear mixed-effects models

### DISCUSSION THREAD

**Discussion by Marci93Bio** [JuliaStats/MixedModels.jl#739] [Ideas]
Dear MixedModels.jl developers,

I was wondering if there is any possibility in the future to include nonlinear mixed-effects models (NLMEs) in the MixedModels.jl package.

Thank you for your time and consideration.

Sincerely,
M.

**Comment by palday** [comment]
Sorry for the delayed response. Currently, nonlinear models aren't on our roadmap. I would be interested in implementing them, but it would be a non trivial undertaking. [Pumas.jl](https://pumas.ai/our-products/products-suite/pumas) (commercial product) may be a viable alternative (not an endorsement).
